templates/systemd-networkd-vxlan-drop-in.conf.j2

```diff
@@ -0,0 +1,2 @@
+[Network]
+VXLAN={{ item.device }}
```

Comment:
Have you tried this on a fresh deployment where this device's main config may not exist yet?

Reply:
Yes. The drop in works nicely. Thanks for the initial suggestion btw :)